frontend/src/components/ReferralCreationModal/ReferralCreationModal.tsx

```diff
@@ -45,17 +57,39 @@ export default function ReferralCreationModal(props: ReferralCreationModalProps
     setActiveStep(activeStep + 1)
   }
 
-  const handleSubmit = () => {
+  const handleSubmit = async () => {
     // TODO: Send post request to backend
-    closeModal()
+    axios({
+      method: 'post',
+      url: '/referral',
+      headers: {
+        'Authorization': localStorage.getItem('authorization')
+      },
+      data: {
+        resumeFileName: resumeFilePath,
+        employeeId,
+        jobPostId: jobPost?.id,
+        description: recommendation,
+        // pronoun
+        email,
+        phone,
+        firstName,
+        lastName,
+      }
+    }).then(res => {
+      closeModal()
```

Comment:
@Danny123234 
The way I have it right now it should close as long as the request is successful. My guess is that the request was passed some invalid data.